### ISSUE TITLE
Moving Head Effect - Reset To Default Button in Status Window is Incorrect

### ISSUE THREAD

**Issue by AzGilrock** [xLightsSequencer/xLights#5061]
The entire concept of the  moving head effect is that each effect can target just a single moving head or a set of heads.  The reset on the Status page is supposed to clear everything out so you can start fresh not apply 0 default to every head cause now that forces every head to have a command.  The status page should be blank after a reset.

**Comment by cybercop23** [comment]
Yes... hmmmm.. hidden... I think there's a resetall function that was called. Maybe it doesn't handle the hidden ones maybe?
